src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2669,11 +2672,16 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_CommandPaletteClosed(const IInspectable& /*sender*/,
                                              const RoutedEventArgs& /*eventArgs*/)
     {
-        // Return focus to the active control
-        if (auto index{ _GetFocusedTabIndex() })
+        // We don't want to set focus on the tab if fly-out is open as it will be closed
+        // TODO: consider checking we are not in the opening state, by hooking both Opening and Open events
```

Comment:
Do we want to address this TODO during this review?

Reply:
@zadjii-msft - don't think so - its highly theoretical.